src/coredump/coredumpctl.c

```diff
@@ -128,11 +131,12 @@ static void help(void) {
                "     --no-pager      Do not pipe output into a pager\n"
                "     --no-legend     Do not print the column headers.\n"
                "  -1                 Show information about most recent entry only\n"
+               "  -S --since=DATE    Only print core dumps since the date\n"
+               "  -U --until=DATE    Only print core dumps until the date\n"
```

Comment:
the help text so far spelt that as "coredump", not "core dump" (i.e. no space), and I figure we should stick to one spelling.